app/qml/map/MapWrapper.qml

```diff
@@ -883,12 +897,26 @@ Item {
   function edit( featurepair ) {
     __activeProject.setActiveLayer( featurepair.layer )
     centerToPair( featurepair )
+    howtoEditingBanner.show()
 
     internal.featurePairToEdit = featurepair
+    internal.startEditingFromScratch = false
```

Comment:
Can you please move this line in the end of `onCanceled` and `onDone` slots of `recordingToolsComponent`?

The thing is you would need to set it to false also in record() and recordInLayer() functions - possibly more in future. If we instead set it to false when editing is finished (after `state = "view"`), we are sure it is not false on the next edit / record / split / whatever :)